SimpleInventoryManagementSystem/Inventory.cs

```diff
@@ -1,15 +1,29 @@
-﻿
-using System;
+﻿using SimpleInventoryManagementSystem;
 
-namespace ISimpleInventoryManagementSystem
+public class Inventory
 {
-    public class Inventory
+    private List<Product> products;
```

Comment:
```suggestion
    private List<Product> _products;
```

https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names

![image](https://github.com/user-attachments/assets/bde2d46a-a825-4bfc-9849-175357f15b8b)